hypothesis-python/src/hypothesis/internal/conjecture/shrinker.py

```diff
@@ -365,10 +366,12 @@ def incorporate_test_data(self, data):
         """Takes a ConjectureData or Overrun object updates the current
         shrink_target if this data represents an improvement over it,
         returning True if it is."""
-        if data is Overrun or data is self.shrink_target:
+        if data.status < Status.VALID or data is self.shrink_target:
```

Comment:
This is locally a functional change, but I'm guessing it's OK because the existing callers require `>= Status.VALID` (or better) in their predicate anyway?

Reply:
Correct, yes.

Reply:
Also this has always been the defined behaviour of the shrinker, we just weren't enforcing it: https://github.com/HypothesisWorks/hypothesis/blob/master/hypothesis-python/src/hypothesis/internal/conjecture/shrinker.py#L119